lib/puppeteer_setup.js

```diff
@@ -0,0 +1,19 @@
+const chalk = require('chalk')
+const puppeteer = require('puppeteer')
+const fs = require('fs')
+const mkdirp = require('mkdirp')
+const os = require('os')
+const path = require('path')
+const app = require('../app/app.js')
+
+const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup')
+
+module.exports = async function () {
+  console.log(chalk.green('\nStart server'))
+  global.__SERVER__ = app.listen(3000)
+  console.log(chalk.green('Setup Puppeteer'))
+  const browser = await puppeteer.launch({args: ['--no-sandbox']})
```

Comment:
Can we comment why we're using --no-sandbox?

Reply:
comment added